cluster/charts/rook-ceph/templates/namespace.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if (ne .Values.clusterNamespace "") }}
+apiVersion: v1
+kind: Namespace
```

Comment:
Based on my experiment with helm v2.16.5, the namespace of `.Release.Namespace` is automatically created by helm, but if we try to separate a ceph cluster namespace by value.yaml, then the ceph cluster namespace is not created by helm. 
Therefore, the namespace should be created in some way.(outside or inside of this helm chart)

The original intention that I add this namespace.yaml is to generate the current `common.yaml` from this helm chart by using `helm template ...`. 
However, that is not possible because helm does not allow to include the namespace resource which is specified by `--namespace` argument.
```
$ helm install --namespace rook-ceph ./cluster/charts/rook-ceph/
Error: release nordic-kudu failed: namespaces "rook-ceph" already exists
```

Therefore, I think it is okay not to create namespace resource in this helm chart(i.e delete namespace.yaml) and just add some document about the procedure of dividing namespaces.

Reply:
I see now that the namespace was intended to be created only for the cluster if it is different from the operator namespace. So helm doesn't allow namespace creation anyway?